include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -24,33 +24,29 @@
 #    error The activated radiation plugin (radiation.param) requires HDF5
 #endif
 
-#include "picongpu/simulation_defines.hpp"
-
-#include "picongpu/traits/SplashToPIC.hpp"
-#include "picongpu/traits/PICToSplash.hpp"
 #include "picongpu/particles/traits/SpeciesEligibleForSolver.hpp"
-
-#include "picongpu/plugins/radiation/Radiation.kernel"
-#include "picongpu/plugins/radiation/ExecuteParticleFilter.hpp"
 #include "picongpu/plugins/ISimulationPlugin.hpp"
 #include "picongpu/plugins/common/stringHelpers.hpp"
+#include "picongpu/plugins/radiation/ExecuteParticleFilter.hpp"
+#include "picongpu/plugins/radiation/Radiation.kernel"
+#include "picongpu/simulation_defines.hpp"
```

Comment:
This `"picongpu/simulation_defines.hpp"` file is an exception and when included (rightfully from this file), has to be included first.

Reply:
I reverted to the same order as `dev` in e99de6a

Reply:
`clang-format, using the formatting codification at https://github.com/ComputationalRadiationPhysics/contributing/tree/master/formatting-tools appears to set `simulation_defines.hpp` not as a first file, so that is a necessary manual correction?

Reply:
PIConGPU is shipping its own clang-format file https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/.clang-format
There is no manual change for the code style required except using "east coast" `const`.

Please use this file, currently, header reordering is disabled in this file because it will break everything. 
This will be fixed hopefully in the future.